Documentation/cubes.rst

```diff
@@ -32,8 +33,10 @@ A standard cube:
 A trasparent cube is a cube without any forwarding capability, such as a network monitor, NAT and a firewall.
 
 A trasparent cube:
```

Comment:
`transparent` misspelled in the document.

Reply:
Thanks Aasif! Now it should be fixed.